recipes-test/ci-all-packages/ci-all-packages.bb

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) Siemens AG, 2022
+#
+# Authors:
+#  Felix Moessbauer <felix.moessbauer@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.
+# See COPYING.MIT file in the top-level directory.
+#
+
+# This is a meta recipe to test the building of all packages
+
+inherit dpkg-raw
```

Comment:
Hmm, do we actually need to build a package for this? As long as we do not build an image and install this as meta package, we may even skip this.

Reply:
Without inheriting something (?), the `DEPENDS` are not picked up.
This package will only end up in the local build tree, which is used by nobody.

But maybe we should rename the recipe to `meta-coral-ci` to avoid name clashes with other layers that include meta-coral.

Reply:
Yeah, would likely still need the base class... Anyway, not that important.